### PR TITLE
Example and script to patch the bug when upgrading 1.1 to 1.2

### PR DESCRIPTION
**Pull request by axsaucedo**
Inlcudes notebook to show how to fix patch

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **axsaucedo**

If they are not already assigned, you can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Sat Jun 27 10:19:31 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2021/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2021 --build=2

**Comment by seldondev** [comment]
Sat Jun 27 10:19:32 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2021/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2021 --build=3

File: doc/source/examples/patch_1_2.nblink [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "path": "../../../examples/models/sklearn_iris/sklearn_iris.ipynb"
```
Comment (cliveseldon) [inline doc/source/examples/patch_1_2.nblink:2]: why is this file named patch but points to sklearn_iris. Same for above but other way round?

Comment (axsaucedo) [reply doc/source/examples/patch_1_2.nblink]: Nice catch it seems I ended up editing the wrong file, I've updated it now

**Comment by seldondev** [comment]
Sat Jun 27 14:33:32 UTC 2020
The logs for [pr-build] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2021/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2021 --build=4

**Comment by seldondev** [comment]
Sat Jun 27 14:33:37 UTC 2020
The logs for [lint] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2021/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2021 --build=5

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/2021#issuecomment-650568715" title="Approved">cliveseldon</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [cliveseldon]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Mon Jul  6 11:39:18 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2021/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2021 --build=1

**Pull request merged**